lfucg-exactions/server/accounts/serializers.py

```diff
@@ -24,21 +23,23 @@ class AccountSerializer(serializers.ModelSerializer):
     address_state_display = serializers.SerializerMethodField(read_only=True)
 
     balance = serializers.SerializerMethodField(read_only=True)
-    credit_availability = serializers.SerializerMethodField(read_only=True)
 
     def get_address_state_display(self, obj):
         return obj.get_address_state_display()
 
     def get_balance(self, obj):
         calculated_balance = calculate_account_balance(obj.id)
-        return '${:,.2f}'.format(calculated_balance)
 
-    def get_credit_availability(self, obj):
-        calculated_balance = calculate_account_balance(obj.id)
         if calculated_balance > 0:
-            return 'Credit Available'
+            return {
+                'balance': '${:,.2f}'.format(calculated_balance),
+                'credit_availability': 'Credit Available'
+            }
         else:
-            return 'No Credit Available'
+            return {
+                'balance': '${:,.2f}'.format(calculated_balance),
```

Comment:
Just curious, could you run into a situation with a negative balance?

Reply:
There are negative balances, they have the format $-500.00